lib/webinspector/index.js

```diff
@@ -7,20 +7,31 @@ import LengthBasedSplitter from '../util/transformer/length-based-splitter';
 
 const WEB_INSPECTOR_SERVICE_NAME = 'com.apple.webinspector';
 
+const PARTIAL_MESSAGE_SUPPORT_DEPRECATION_VERSION = 11;
+
 class WebInspectorService {
 
-  constructor (socketClient) {
+  constructor (version, socketClient) {
     this._socketClient = socketClient;
 
-    // 1MB as buffer for bulding webinspector full messages. We can increase the value if more buffer is needed
-    this._decoder = new WebInspectorDecoder(1024 * 1024);
-    this._plistDecoder = new PlistServiceDecoder();
-    this._splitter = new LengthBasedSplitter(false, 1000000, 0, 4, 4);
-    this._socketClient.pipe(this._splitter).pipe(this._plistDecoder).pipe(this._decoder);
-
-    this._encoder = new WebInspectorEncoder();
-    this._plistEncoder = new PlistServiceEncoder();
-    this._encoder.pipe(this._plistEncoder).pipe(this._socketClient);
+    if (version.major < PARTIAL_MESSAGE_SUPPORT_DEPRECATION_VERSION) {
+      // 1MB as buffer for bulding webinspector full messages. We can increase the value if more buffer is needed
+      this._decoder = new WebInspectorDecoder(1024 * 1024);
```

Comment:
I'd rather put these values to constants

Reply:
I will address this later on